recipes/domclust/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: domclust 
+  version: "1.2.7l"
+
+about:
+  home: http://mbgd.genome.ad.jp/domclust/
+  license: 
```

Comment:
As far as I can see the original developers have not supplied one. What is the best practice in such a case?